packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -118,6 +118,21 @@ class ProjectUpdate(InputSchema):
     quality: dict[str, Any] = FieldNotRequired()
 
 
+ProjectName: TypeAlias = IDStr
+ProjectDescription: TypeAlias = IDStr
+
+
+class ProjectPatch(InputSchema):
+    name: ProjectName = Field(None)
```

Comment:
So here ```name``` can be either ProjectName or None since you set Field(None) right?
How come your linter is not complaining about this and all the fields below?

```suggestion
    name: ProjectName | None = None
```

Reply:
This is a way how you define in Pydantic non-nullable optional fields.

Reply:
did you not forget to use FieldNotRequired here?